libpromises/cf3lex.l

```diff
@@ -140,14 +140,15 @@ promise_type   [a-zA-Z_]+:
                           P.current_line = xstrdup(yytext+1);
                           ParserDebug("L:line %s\n", P.current_line);
                           P.line_no++;
-                          P.line_pos = 0;
+                          P.line_pos = 1;
                           // push back on stack and skip first char
                           yyless(1);
                       }
 
 {macro_if_version}    {
-                        if (  P.line_pos != 0 )
+                        if (  P.line_pos != 1 )
                         {
+                             printf("line %d, pos = %d\n", P.line_no, P.line_pos);
```

Comment:
Is ``printf()`` really a good thing to call here?

Reply:
sorry is a left over.  Good catch.